cert/aziot-certd-config/src/lib.rs

```diff
@@ -247,6 +251,23 @@ impl std::convert::TryFrom<&CertSubject> for openssl::x509::X509Name {
     }
 }
 
+pub fn deserialize_url_map_check_https<'de, D>(de: D) -> Result<BTreeMap<String, Url>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let map = BTreeMap::<String, Url>::deserialize(de)?;
+    for url in map.values() {
+        if url.scheme() == "http" {
+            eprintln!(
+                "Warning: EST server URL {:?} is configured with unencrypted HTTP, which may expose device to man-in-the-middle attacks.\
```

Comment:
One space is needed for the `\`. ie `attacks. \`

Reply:
Ah yes I forgot to add the space in this println. Thank you for the catch